src/main/java/Task/ToDo.java

```diff
@@ -0,0 +1,43 @@
+package Task;
+
+import java.util.Objects;
+
+/**
+ * Represents a to-do task without any specific time constraints.
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructs a new ToDo task with the specified description.
+     *
+     * @param description The description of the to-do task.
+     */
+    public ToDo(String description) {
+        super(description);
+        this.type = "[T]";
+    }
+
+    /**
+     * Loads a ToDo task from a formatted string array obtained from splicing the save file
+     * during Storage's load method.
+     *
+     * @param properties The string array containing the task properties.
+     */
+    public static void load(String[] properties) {
+        ToDo newTodo = new ToDo(properties[2]);
+        if (Objects.equals(properties[1], "1")) {
```

Comment:
out of curiosity, why use the objects.equals methods, and not just a `"1".equals(properties[1])`